src/dataexplorer/dataexplorer-application/src/components/PatientInfo/PatientInfo.tsx

```diff
@@ -180,33 +178,78 @@ const displayPatientInfo = (
     handleClick: any,
     dataSource: any,
 ) => {
+    const calcAge = () => {
+        let birthday = patient.birthDate;
+        let patientYear = parseInt(birthday.substring(0, 4));
+        let patientMonth = dayjs(birthday.substring(5, 7))
+            .locale('nb')
+            .format('MMMM');
+        let patientDay = parseInt(birthday.substring(8, 10));
+        let actualAge = dayjs().diff(birthday, 'year');
+
+        return (
+            actualAge +
+            ' år, født ' +
+            patientDay +
+            '. ' +
+            patientMonth +
+            ' ' +
+            patientYear
+        );
+    };
+    const { Title } = Typography;
     const name = patient.name[0].given[0] + ' ' + patient.name[0].family;
     return (
         <>
             <Row gutter={[1, 40]} justify="center">
                 <Col span={12}>
                     <Card
+                        key={patient.id}
                         className="patient-card"
+                        title={<Title level={4}>{name}</Title>}
+                        headStyle={{
+                            backgroundColor: '#f7fbff',
+                            display: 'flex',
+                            justifyContent: 'center',
+                            marginBottom: 0,
+                        }}
                         type="inner"
                         hoverable
-                        key={patient.id}
-                        title={name}
+                        bordered
                     >
                         <div className="info-container">
                             <div className="info-left">
-                                <h4>Pnr:</h4>
-                                <span>{patient.id}</span>
+                                <h4>Personnummer:</h4>
+                                <span>{patient.identifier[0].value}</span>
                                 <h4>Kjønn:</h4>
-                                <span>{patient.gender}</span>
-                                <h4>Fødselsdato:</h4>
-                                <span>{patient.birthDate}</span>
+                                <span>
+                                    {patient.gender === 'male'
+                                        ? 'Mann'
+                                        : 'Kvinne'}
```

Comment:
Is the gender necessarily either male or female?

Reply:
Fixed